Resources/Locale/en-US/job/job-description.ftl

```diff
@@ -45,8 +45,4 @@ job-description-security = Catch criminals and enemies of the station, enforce t
 job-description-serviceworker = Learn the basics of bartending, cooking, and growing plants.
 job-description-visitor = Enjoy your visit to the station.
 job-description-warden = Patrol the security department, ensure that no one is stealing from the armory, and make sure that all prisoners are processed and let out when their time is up.
-job-description-zookeeper = Put on a joyful display of cute animals and space carps for all the crew to see. Currently unavailable.
-job-description-senior-engineer = Teach new engineers the basics of the station's engine, repairing, atmospherics and power.
-job-description-senior-researcher = Teach new scientists the basics of item printing, artifact research and anomalous objects.
-job-description-senior-physician = Teach new medics the basics of tending to the wounded, chemistry, diagnosing the diseased and disposing of the dead.
-job-description-senior-officer = Teach new officers the basics of searches, performing arrests, prison times and how to properly shoot a firearm.
+job-description-zookeeper = Put on a joyful display of cute animals and space carps for all the crew to see. Currently available on Gemini Station.
```

Comment:
Should maybe change to `Currently available on Submarine Station.`

Reply:
probably a bad idea to mention any station, considering other maps might add the role

Reply:
Either way. Can always update if another station adds it but leaving it out is less future edits. 🤷‍♂️